index.html

```diff
@@ -1,13 +1,132 @@
+            </h1>
+            <p class="description">days of intense
+                cultural artistic activity with dialogue
+                increasingly
+                deep among the peoples, becoming one of
+                major cultural events
+                importance in the country, due to its intensity
+                and diversity of cultural artistic offer.</p>
+            <p class="date">2022.04.13 (wed) ~ 16 (sat)</p>
+            <p class="description2">@Culture's house Tarija</p>   
+    </section>
+    </div>
+  <section class="program" id="program">
+      <h2>Main Program</h2>
+      <div class="red-line"></div>
+      
+      <div class="programs">
+      <article>
+        <img src="./img/program_icon_01.png" alt="program1">
+        <h3>Literature</h3>
+        <p>Qui officia deserunt mollit anim id est laborum. Totam rem aperiam. Nihil molestiae consequatur, vel illum qui dolorem eum. 
+            Totam rem aperiam. Cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia.</p>
+        </article>
+
+      <article>  
+        <img src="./img/program_icon_02.png" alt="program2">       
+        <h3>Exhibition</h3>
+        <p>Qui officia deserunt mollit anim id est laborum. Do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+            At vero eos et accusamus. Nihil molestiae consequatur, vel illum qui dolorem eum. Corrupti quos dolores et quas molestias excepturi sint occaecati.</p>
+      </article>
+      <article>
+        <img src="./img/program_icon_03.png" alt="program3">
+        <h3>Theater</h3>
+        <p>Fugiat quo voluptas nulla pariatur? Do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+            Excepteur sint occaecat cupidatat non proident, sunt in culpa. Do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+      </article>
+      <article>
+        <img src="./img/program_icon_04.png" alt="program4">
+          <h3>Sculpture</h3>
+          <p>Esse cillum dolore eu fugiat nulla pariatur. Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+               Sed ut perspiciatis unde omnis iste natus error sit voluptatem. Nihil molestiae consequatur, vel illum qui dolorem eum.</p>
+      </article>
+      <article>
+        <img src="./img/program_icon_05.png" alt="program5">
+          <h3>Orchestra</h3>
+          <p>Corrupti quos dolores et quas molestias excepturi sint occaecati. Ut aut reiciendis voluptatibus maiores alias consequatur aut perferendis doloribus asperiores repellat.
+               Nihil molestiae consequatur, vel illum qui dolorem eum.</p>
+      </article>
+    </div>
+      <a href="#">SEE THE WHOLE PROGRAM</a>
+      <button type="submit">Join ACF Tarija 2022</button>
+  </section>
+  <section class="featured" id="speakers">
+       <h2>Featured Art Masters</h2><div class="red-line"></div>
+       <div class="art" id="speaker"></div>
+  </section>
+<section class="partners" id="sponsor">
```

Comment:
- [ ] [Optional] The images in this section should be centered with no background or dark background, please consider making the changes so the design is as close to the provided one as possible

|Your design|CC design|
|---|---|
|![Screenshot_20220519_212441](https://user-images.githubusercontent.com/25227022/169398263-62a246a8-4ec6-47d8-966c-71c8540f5fe9.png)|![Screenshot_20220519_212123](https://user-images.githubusercontent.com/25227022/169398161-833c6b0e-d99f-457f-9051-bbc6be76cfa6.png)|